site/source/user-manual/reset-password.rst

```diff
@@ -0,0 +1,46 @@
+.. _reset-password:
+
+===================
+Reset Your Password
+===================
+
+.. contents::
+  :depth: 2
+  :local:
+
+#. Obtain the latest copy of EmbassyOS with your product key included, by `downloading it from our image server <https://images.start9.com/>`_.
+#. Flash the downloaded image to your microSD card, using `this guide <../flashing>`_.
+#. Insert the microSD card into your Embassy and power it on.
+
+    .. note:: Embassy will power up and then initialize.  Once complete, you will hear a _bep_ to indicate it is initialized and then a _chime_ sound to indicate it is online.
+
+#. On your computer, open up a browser and go to http://embassy.local.
+#. When prompted, enter your Product Key:
```

Comment:
We may want to say "Product Key (if asked)" - since product keys will no longer exist if they get a fresh image

Reply:
or this one